stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -223,6 +223,10 @@ lazy_static! {
     pub static ref NAKAMOTO_CHAINSTATE_SCHEMA_2: Vec<String> = vec![
     NAKAMOTO_TENURES_SCHEMA_2.into(),
     r#"
+    ALTER TABLE nakamoto_block_headers
+      ADD COLUMN timestamp INTEGER NOT NULL;
```

Comment:
You're going to need to provide a default value if you're going to add a non-null column (see section 4 here: https://www.sqlite.org/lang_altertable.html)

Reply:
I think this is okay since we knew that the table is empty (changed before 3.0 starts). I tested this out.